app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt

```diff
@@ -576,6 +581,25 @@ object TelemetryWrapper {
         TelemetryEvent.create(Category.ACTION, Method.CLICK, Object.MENU, Value.RELOAD).queue();
     }
 
+    @JvmStatic
+    fun SSLError(fromPage: Boolean, error: SslError) {
+        val primaryErrorMessage:String
+        when (error.primaryError) {
```

Comment:
Nice use of kotlin state. Optionally include a link to the API here that lists these ssl errors.